lxc/lxc-openvdc/lxc-openvdc.go

```diff
@@ -0,0 +1,216 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/mholt/archiver"
+	"github.com/pkg/errors"
+)
+
+var lxcPath string
+var cacheFolderPath string
+var containerPath string
+var containerName string
+var rootfsPath string
+var imgPath string
+
+func main() {
+
+	_dist := flag.String("dist", "centos", "Name of the distribution")
+	_release := flag.String("release", "7", "Release name/version")
+	_arch := flag.String("arch", "amd64", "Container architecture")
+	_rootfs := flag.String("rootfs", "", "Rootfs path")
+	_containerName := flag.String("name", "", "Container name")
+	_containerPath := flag.String("path", "", "Container path")
+	_errorLogPath := flag.String("error-log-path", "/etc/openvdc/", "Error log path")
```

Comment:
``/etc`` is not the location to place log file. In addition, streaming the log to syslog or journald is better.